test.py

```diff
@@ -0,0 +1,157 @@
+import json
+import logging
+import types
+import webapp2
```

Comment:
by the by, I prefer to pile all the imports on the same line because nobody's paying per line of code, and it's not like there's real logic or creativity here - and (sigh) TBH, python should really allow you to auto-import common libraries from a config file, instead of burying every .py file with bullshit imports.